spec/factories/attendees.rb

```diff
@@ -0,0 +1,15 @@
+FactoryGirl.define do
+  factory :attendee do
+    id 1
+    user_id 1
+    event_id  1
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.